dotCMS/src/main/java/com/dotcms/security/apps/SecretsKeyStoreHelper.java

```diff
@@ -374,89 +334,20 @@ protected String digest(final String val) {
      * @return
      */
     @VisibleForTesting
-    protected char[] decrypt(final char[] encryptedString) {
+    char[] decrypt(final char[] encryptedString) {
```

Comment:
Codacy found an issue: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6286472)